okapi-core/src/main/java/org/folio/okapi/service/impl/DockerModuleHandle.java

```diff
@@ -37,6 +37,10 @@
 @java.lang.SuppressWarnings({"squid:S1192"})
 public class DockerModuleHandle implements ModuleHandle {
 
+  static final String DOCKER_REGISTRIES_EMPTY_LIST =
+      "dockerRegistries=[] contains no registry and disables docker pull. "
+          + "For unauthenticated pull from DockerHub use [ {} ].";
+
   private final Logger logger;
```

Comment:
I think it's better to assign a message for this by adding an entry in okapi-core/src/main/resources/infra-messages/Messages_en.properties

Reply:
https://github.com/folio-org/okapi/tree/master/okapi-core/src/main/resources/infra-messages hasn't been used since 2018. And there isn't any translation yet. I would rather propose to remove all infra-messages to improve code readability.

Reply:
Are you kidding?

Reply:
FWIW, let's must merge this now and have this discussion later.. The messages returned IMHO should be accommodated with the message ID .. Eg (OKAPI-12345) messsage .. It serves as a nice unique way of telling what has returned the message. Of course, the log might have backtraces and all.. But for a system user, it's nice to be able to spot WHAT has actually generated some message .. Okapi? RMB ? Or code from a module?